app/[locale]/providers.tsx

```diff
@@ -24,7 +25,7 @@ function LocaleDetector() {
 export const Providers = ({ children, locale }: PropsWithChildren<{ locale: string }>) => {
   return (
     <>
-      <AnalyticsProvider />
+      {env.NEXT_PUBLIC_OPENPANEL_ENABLED && <AnalyticsProvider />}
```

Comment:
```suggestion
      <AnalyticsProvider /> 
```

if you accept my (below) modification